src/chainstate/stacks/index/node.rs

```diff
@@ -1779,15 +1742,15 @@ mod test {
             TrieNodeID::Node256
         ];
         // ptrs (256)
+
+        let pointer_back_block_bytes = [0; 32];
         for i in 0..255 {
-            node256_bytes.append(&mut vec![
-                TrieNodeID::Node256, i as u8, 0, 0, 0, 0
-            ]);
+            node256_bytes.extend_from_slice(&[TrieNodeID::Node256, i]);
```

Comment:
Same here.

Reply:
I think I addressed in b8594a21 -- the version in `develop` at the moment uses a for-loop to add in the pointer vectors, and I do that in b8594a21 too. Let me know if you think I should expand out the loop.